samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/KMLLayers.razor

```diff
@@ -0,0 +1,53 @@
+﻿@page "/kmllayers"
+
+<h3>Kml Layers</h3>
+
+<div class="links-div">
+    <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-KMLLayer.html">ArcGIS API for JavaScript</a>
+</div>
+<Label>Add the sample KML Layer using a URL to see major earthquakes for latest 30 days from USGS to the map database :</Label>
+<InputText @bind-Value="_kmlLayerUrl"></InputText>
+<button disabled="@(!_mapRendered)" @onclick="(()=>AddRemoveKMLLayer())">Add or Remove KML Layer in code</button>
+<button disabled="@(!_mapRendered)" @onclick="(()=>_markup = !_markup)">Add new KML Layer in Markup</button>
+
+
+<MapView @ref="_view" class="map-view" OnMapRendered="OnMapRendered" Scale="0">
+    <Map ArcGISDefaultBasemap="dark-gray-vector">
+        @if (_markup)
+        {
+            <KMLLayer Url="https://earthquake.usgs.gov/fdsnws/event/1/query?format=kml&minmagnitude=5.8" Title="Major EarthQuakes Query Result"></KMLLayer>
```

Comment:
You have this url string as a field already, you could just set it here with `<KMLLayer Url="@_kmlLayerUrl" Title="Major Earthquakes Query Result" />`

Reply:
corrected